### DISCUSSION TITLE
Left/right arrow keys to collapse/expand nodes/folders in search results and browsed folders

### DISCUSSION THREAD

**Discussion by ReinyLegit** [nicotine-plus/nicotine-plus#2307] [Ideas]
This works in SoulseekQT but not in Nicotine+. Is there anything that prevents implementing such a feature?

**Comment by slook** [comment]
It is implemented in Browse Shares but not yet in Search Files, Downloads, Uploads views.

This request is blocked until we've ported all lists to the new TreeView class.
